web/src/features/percentileCalculator/components/TimeRangeSlider.tsx

```diff
@@ -17,8 +17,7 @@ interface Props {
 }
 
 const MIN_YEARS = 0
-
-const MAX_YEARS = new Date().getFullYear() - 1970
+const MAX_YEARS = 50
```

Comment:
Why this change? Won't this limit the percentile calculator to 2020? If we are limiting the percentile  calculator to 2020 (which I guess, since we only have data for that - we should) - I'd rather we be explicit about it and use 2020 in the code instead of 50, which seems arbitrary.